src/transformStateWithClones.js

```diff
@@ -65,7 +65,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  const clone = { ...state };
+
+  for (let i = 0; i < transforms.length; i++) {
+    const operation = transforms[i].operation;
+
+    switch (operation) {
+      case 'clear':
+        for (const property in clone) {
```

Comment:
property consists of key and value
here you have a key

```suggestion
        for (const key in clone) {
```